FH.podspec

```diff
@@ -14,4 +14,5 @@ Pod::Spec.new do |s|
   s.libraries = 'xml2', 'z'
   s.dependency 'ASIHTTPRequest/Core', '1.8.2'
   s.dependency 'Reachability', '3.2'
+  s.dependency 'AeroGear-Push', '1.0.0'
```

Comment:
I think adding ```s.dependency``` in the library itself has flows:

a) it implies the FH library uses the AeroGear-Push internally to function, which afaik that is not the case.
b) breaks the ```FH.framework``` binary framework distribution(not cocoapods) since the headers of the AeroGearPush library are not exposed, which is correct.

Although I understand it's convenient for the cocoapods users since they don't require adding 'AeroGearPush' in their Podfile, the usage flow is wrong. It is the same thing as the following case:

library A depends on AFNetworking and because of that I can use AFNetworking in my programs. This is not encouraged, since what happens later on if library A decides to depend on another library. All the ```#import``` statements are now causing compiler errors.

If a lighter integration is the goal, I believe this should be visible upfront and comply with the dependency management rules, both binary framework and cocoapods usage. That is:

a) binary FH.framework (as it is currently) and a binary AeroGearPush.framework (as it is currently). Users should be able to drop those two frameworks in their project and start working. The same with cocoapods, users should specify both e.g
```
pod FH
pod AeroGear-Push
```

Long term, and if deeper integration is the goal:
Add the dependency in AeroGearPush and on FH framework create a wrapper class that invokes the AeroGearPush library. Users are hidden from the fact that they are using AeroGearPush but instead rely on what the ```FHPush.h``` file expose. This will allow to add features and proper align with the rules of FH. One that I can think right now, should the user have initialised the framework before registering with UPS. Currently they are independent flows

@corinnekrych @matzew let me know what you think.